src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2056,7 +2056,7 @@ NetworkOPsImp::pubServer()
                     f.em->openLedgerFeeLevel,
                     f.loadBaseServer,
                     f.em->referenceFeeLevel)
-                    .second);
```

Comment:
Are cases like this guaranteed to have a value? Is that why the old code didn't check for success? If so, then I think the new code should just use [`operator*`](https://en.cppreference.com/w/cpp/utility/optional/operator*).

Reply:
Cases like this are not guaranteed to have a value.  When the old behavior overflowed, it passed back an overflow flag and the value `ripple::muldiv_max`.  The client level code (in this example) didn't check the overflow flag and just took the value.  In this rewrite, the client code is forced to be explicit that it is using `ripple::muldiv_max` on overflow.

In any case the behavior is unchanged.